airflow/executors/base_executor.py

```diff
@@ -52,26 +170,30 @@ class BaseExecutor(LoggingMixin):
     """
     def __init__(self, parallelism: int = PARALLELISM):
         super().__init__()
-        self.parallelism: int = parallelism
-        self.queued_tasks: OrderedDict[TaskInstanceKeyType, QueuedTaskInstanceType] \
-            = OrderedDict()
-        self.running: Set[TaskInstanceKeyType] = set()
-        self.event_buffer: Dict[TaskInstanceKeyType, Optional[str]] = {}
+        self._parallelism: int = parallelism
```

Comment:
Why?

Reply:
Just to not access it directly (encapsulation). But true. Since it's python and it's a number, I can leave it as it is.